content/php/concepts/arrays/terms/array-search/array-search.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'array_search()'
+Description: 'Searches an array for a given value and returns the first matching key for that value.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_seach()`** function searches an array for a given value and returns the first matching key for that value.
+
+## Syntax
+
+```pseudo
+array_search(value, array, strict)
+```
+
+- `value`: The value to search for.
+> **Note:** String values are case-sensitive for this function.
```

Comment:
I would suggest moving the note after all the parameters

Reply:
Fixed!